src/routes/protectedRoute/index.tsx

```diff
@@ -1,11 +1,8 @@
 import React, { useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
+import { IProtectedRouteProps } from '../../interfaces/interface-router';
 
-interface ProtectedRouteProps {
-  children: React.ReactNode;
-}
-
-export const ProtectedRoute: React.FC<ProtectedRouteProps> = ({ children }) => {
+export const ProtectedRoute: React.FC<IProtectedRouteProps> = ({ children }) => {
```

Comment:
Como eu disse no outro comentário, essa interface poderia ser subsituída por 
```suggestion
export const ProtectedRoute: React.FC<React.PropsWithChildren> = ({ children }) => {
```

Reply:
Alteração realizada com sucesso.